crates/e2e/src/setup/services.rs

```diff
@@ -193,13 +193,15 @@ impl<'a> Services<'a> {
     }
 
     /// Starts a basic version of the protocol with a single legacy solver and
-    /// quoter.
+    /// quoter. Optionally starts a baseline solver and uses it for price
+    /// estimation.
     pub async fn start_protocol_legacy_solver(
```

Comment:
This function is not used in our code base. Given that we are transitioning towards dropping the legacy API are you still using this function?
If not we should just delete it.

Reply:
You're right, thanks! We don't need it anymore. Deleted in my new commit.